scripts/generate-doc.sh

```diff
@@ -76,27 +76,18 @@ if [ ! -f "docs/resources/${resource_name}.md" ]; then
     echo "Error: We cannot find the documentation file for the resource ${resource_name}.md"
     echo "Please, make sure to include the resource template under templates/resources/${resource_name}.md.tmpl"
     printf "Skipping this step: We assume that only a data source is being generated.\n\n"
-else
```

Comment:
files are already in correct new folder with new name

Reply:
Nice that you contemplated adjusting this script. I am however concerned this will now not work as expected because `tfplugindocs generate` generates automated docs for all registered resources in `docs` folder, and then we are moving the specific resource docs and deleting all the rest. Would verify if my assumption is correct and look into options for preserving existing behaviour of only updating specific resource/data source documentation.

Reply:
thanks @AgustinBettati , you're right all files are overwritten not only the specified resource. I think this commit should fix it: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2425/commits/4a3006cd969de35e1124bb73b32f3c6293a05dc6